app/services/billable_metrics/aggregation_factory.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module BillableMetrics
+  class AggregationFactory
+    def self.new_instance(charge:, current_usage: false, **attributes)
+      aggregator_class(charge, current_usage).new(
+        billable_metric: charge.billable_metric,
+        **attributes
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)